servant-client/test/Servant/ClientSpec.hs

```diff
@@ -120,6 +119,53 @@ type Api =
 api :: Proxy Api
 api = Proxy
 
+getGet               :: SCR.ClientM Person
+getDeleteEmpty       :: SCR.ClientM NoContent
+getCapture           :: String
+                     -> SCR.ClientM Person
+getBody              :: Person
+                     -> SCR.ClientM Person
+getQueryParam        :: Maybe String
+                     -> SCR.ClientM Person
+getQueryParams       :: [String]
+                     -> SCR.ClientM [Person]
+getQueryFlag         :: Bool
+                     -> SCR.ClientM Bool
+getRawSuccess        :: HTTP.Method
+                     -> SCR.ClientM ( Int
+                                    , BS.ByteString
+                                    , MediaType
+                                    , [HTTP.Header]
+                                    , C.Response BS.ByteString )
+getRawFailure        :: HTTP.Method
+                     -> SCR.ClientM ( Int
+                                    , BS.ByteString
+                                    , MediaType
+                                    , [HTTP.Header]
+                                    , C.Response BS.ByteString )
+getMultiple          :: String
+                     -> Maybe Int
+                     -> Bool
+                     -> [(String, [Rational])]
+                     -> SCR.ClientM ( String
+                                    , Maybe Int
+                                    , Bool
+                                    , [(String, [Rational])] )
+getRespHeaders       :: SCR.ClientM (Headers TestHeaders Bool)
+getDeleteContentType :: SCR.ClientM NoContent
+getGet
+    :<|> getDeleteEmpty
+    :<|> getCapture
+    :<|> getBody
+    :<|> getQueryParam
+    :<|> getQueryParams
+    :<|> getQueryFlag
+    :<|> getRawSuccess
+    :<|> getRawFailure
+    :<|> getMultiple
+    :<|> getRespHeaders
+    :<|> getDeleteContentType = client api
+
```

Comment:
Personally I wouldn't align types (or code in general) at all. I would go with two spaces indentation (although I don't have anything against 4). So I would prefer e.g.

``` haskell
getRawFailure :: HTTP.Method
  -> SCR.ClientM (Int, BS.ByteString, MediaType, [HTTP.Header], C.Response BS.ByteString)
```

Mainly because of this: http://blog.madewithlove.be/post/code-style-options-for-cleaner-diffs/, Tip 2.

`servant` code uses alignment in lots (most?) places, so this is really just my 2 cents.